packages/web/config.ts

```diff
@@ -17,7 +17,15 @@ export const CONFIG = {
     process.env.NEXT_BRIGHTID_NODE_URL || 'http:%2f%2fnode.brightid.org',
   publicURL: process.env.NEXT_PUBLIC_FRONTEND_URL,
   gaId: process.env.NEXT_PUBLIC_GA4_ID,
-  clarityId: process.env.NEXT_CLARITY_ID,
+  honeybadgerApiKey:
+    process.env.NEXT_PUBLIC_HONEYBADGER_API_KEY ||
+    'hbp_Z4027J7qYTktQkqvUHcOHJqIbpjpWD30A1kd',
```

Comment:
Should these be secrets?

Reply:
Probably should be. I'll change the keys and remove from config.

Reply:
> Should these be secrets?

Do you have access to the cloud account?

Reply:
no